lib/conjur/command/users.rb

```diff
@@ -136,7 +136,7 @@ class Conjur::Command::Users < Conjur::Command
       c.flag [:p,:password]
 
       c.action do |global_options,options,args|
-        username, password = Conjur::Authn.read_credentials
+        username, password = Conjur::Authn.get_credentials
```

Comment:
Wouldn't that mean that if the user tried to change his password, but wasn't logged in, he'd get an error? Is that the behavior we want?